README.md

```diff
@@ -55,16 +67,23 @@ make sanity
 
 ### Running a node
 
-Once the software is built you'll find binaries in `${GOPATH}/bin`, and a data directory will be initialized at `~/.algorand`. Start your node with `${GOPATH}/bin/goal node start -d ~/.algorand`, use `${GOPATH}/bin/carpenter -d ~/.algorand` to see activity. Refer to the [developer website][developer site url] for how to use the different tools.
+Once the software is built you'll find binaries in `${GOPATH}/bin`, and a data
+directory will be initialized at `~/.algorand`. Start your node with
+`${GOPATH}/bin/goal node start -d ~/.algorand`, use `${GOPATH}/bin/carpenter -d
+~/.algorand` to see activity. Refer to the [developer website][developer site
+url] for how to use the different tools.
 
-#### Providing your own data directory
-You can run a node out of other directories than `~/.algorand` and join networks other than mainnet. Just make a new directory and copy into it the `genesis.json` file for the network. For example:
+#### Providing your own data directory You can run a node out of other
```

Comment:
I think there should be a line break after "data directory"

Reply:
yes! good catch.